workflow/scripts/cnv_report.py

```diff
@@ -175,7 +171,19 @@ def create_tsv_report(
     in_amp = snakemake.input.amplifications
     amp_cn_limit = snakemake.params.call_small_amplifications_cn_limit
     out_tsv = snakemake.output.tsv
+    out_additional_only = open(snakemake.output.tsv_additional_only, "w")
     del_1p19q_cn = snakemake.params.del_1p19q_cn_limit
     del_1p19q_chr_arm_fraction = snakemake.params.del_1p19q_chr_arm_fraction
     TC = float(snakemake.params.tc)
-    create_tsv_report(in_vcfs, in_org_vcfs, in_del, in_amp, amp_cn_limit, out_tsv, del_1p19q_cn, del_1p19q_chr_arm_fraction, TC)
+    create_tsv_report(
+        in_vcfs,
+        in_org_vcfs,
+        in_del,
+        in_amp,
+        amp_cn_limit,
+        out_tsv,
+        out_additional_only,
+        del_1p19q_cn,
+        del_1p19q_chr_arm_fraction,
+        TC,
+    )
```

Comment:
Nitpicking, bu perhaps wrapping `create_tsv_report` in a `with open(...)` statement. This works though, so up to you.